### PR TITLE
Bump traefik to v1.2.2

### PR DESCRIPTION
**Pull request by emilevauge**
Hello,

This PR updates traefik to v1.2.2
/cc @vdemeester

Cheers

### CODE REVIEW

File: library/traefik [hunk 1]
```diff
@@ -2,16 +2,16 @@ Maintainers: Emile Vauge <emile@vauge.com> (@emilevauge),
...
+GitCommit: 54df16e9292b5ea7257d5a537653b4b4ac8d8ddf
 
-Tags: v1.2.1-alpine, 1.2.1-alpine, v1.2-alpine, 1.2-alpine, morbier-alpine
-GitCommit: 5444b825df0cd3884956e5a5bf19d7d240ccb056
+Tags: v1.2.2-alpine, 1.2.2-alpine, v1.2-alpine, 1.2-alpine, morbier-alpine
+GitCommit: 54df16e9292b5ea7257d5a537653b4b4ac8d8ddf
 Directory: alpine
 
-Tags: v1.1.2, 1.1.2, camembert
+Tags: v1.1.2, 1.1.2, v1.1, 1.1,, camembert
```
Comment (tianon) [inline library/traefik]: This `,,` should just be `,` :sweat_smile:

Comment (emilevauge) [inline library/traefik]: OMG...

Comment (emilevauge) [inline library/traefik]: Fixed @tianon ^_^

**Comment by tianon** [comment]
Diff:

<details>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 7210c73..6f25a6d 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -1,17 +1,21 @@
+traefik:1.1
+traefik:1.1-alpine
 traefik:1.1.2
 traefik:1.1.2-alpine
 traefik:1.2
 traefik:1.2-alpine
-traefik:1.2.1
-traefik:1.2.1-alpine
+traefik:1.2.2
+traefik:1.2.2-alpine
 traefik:camembert
 traefik:camembert-alpine
 traefik:latest
 traefik:morbier
 traefik:morbier-alpine
+traefik:v1.1
+traefik:v1.1-alpine
 traefik:v1.1.2
 traefik:v1.1.2-alpine
 traefik:v1.2
 traefik:v1.2-alpine
-traefik:v1.2.1
-traefik:v1.2.1-alpine
+traefik:v1.2.2
+traefik:v1.2.2-alpine
diff --git a/traefik_latest/Dockerfile b/traefik_latest/Dockerfile
index c1c3fd0..d3b5764 100644
--- a/traefik_latest/Dockerfile
+++ b/traefik_latest/Dockerfile
@@ -9,5 +9,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \    
-      org.label-schema.version="v1.2.1" \
+      org.label-schema.version="v1.2.2" \
       org.label-schema.docker.schema-version="1.0"
\ No newline at end of file
diff --git a/traefik_latest/traefik b/traefik_latest/traefik
index 941c069..8eb21f4 100755
Binary files a/traefik_latest/traefik and b/traefik_latest/traefik differ
diff --git a/traefik_morbier-alpine/Dockerfile b/traefik_morbier-alpine/Dockerfile
index 7e234e8..b146a47 100644
--- a/traefik_morbier-alpine/Dockerfile
+++ b/traefik_morbier-alpine/Dockerfile
@@ -13,5 +13,5 @@ LABEL org.label-schema.vendor="Containous" \
       org.label-schema.url="https://traefik.io" \
       org.label-schema.name="Traefik" \
       org.label-schema.description="A modern reverse-proxy" \    
-      org.label-schema.version="v1.2.1" \
+      org.label-schema.version="v1.2.2" \
       org.label-schema.docker.schema-version="1.0"
\ No newline at end of file
diff --git a/traefik_morbier-alpine/traefik b/traefik_morbier-alpine/traefik
index 941c069..8eb21f4 100755
Binary files a/traefik_morbier-alpine/traefik and b/traefik_morbier-alpine/traefik differ
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #2849; fffcbcd3 (`traefik`):

```console

$ bashbrew build traefik:v1.2.2
Building bashbrew/cache:1cbc9337af95465ee73e50c6b76edbd0100df9e7683868afc5ad4621e91f2618 (traefik:v1.2.2)
Tagging traefik:v1.2.2
Tagging traefik:1.2.2
Tagging traefik:v1.2
Tagging traefik:1.2
Tagging traefik:morbier
Tagging traefik:latest

$ test/run.sh traefik:v1.2.2
testing traefik:v1.2.2
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.2.2-alpine
Building bashbrew/cache:5e0f4194f882beca23a4da517beeb13644dbef5580c55cd6d02e773e9c58a132 (traefik:v1.2.2-alpine)
Tagging traefik:v1.2.2-alpine
Tagging traefik:1.2.2-alpine
Tagging traefik:v1.2-alpine
Tagging traefik:1.2-alpine
Tagging traefik:morbier-alpine

$ test/run.sh traefik:v1.2.2-alpine
testing traefik:v1.2.2-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...passed


$ bashbrew build traefik:v1.1.2
Using bashbrew/cache:8f13e7a5a0f7f7cbbdb6db73a3fa1f1912ea85dd7f881df215a85148fc7a3c63 (traefik:v1.1.2)
Tagging traefik:v1.1.2
Tagging traefik:1.1.2
Tagging traefik:v1.1
Tagging traefik:1.1
Tagging traefik:camembert

$ test/run.sh traefik:v1.1.2
testing traefik:v1.1.2
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...image does not appear to contain "echo" -- assuming single-binary image
passed


$ bashbrew build traefik:v1.1.2-alpine
Using bashbrew/cache:69c2e81e793fa918f645687b924a583ad83d79cc5a0b9f018ac8b41218f8ee41 (traefik:v1.1.2-alpine)
Tagging traefik:v1.1.2-alpine
Tagging traefik:1.1.2-alpine
Tagging traefik:v1.1-alpine
Tagging traefik:1.1-alpine
Tagging traefik:camembert-alpine

$ test/run.sh traefik:v1.1.2-alpine
testing traefik:v1.1.2-alpine
	'cve-2014--shellshock' [1/2]...passed
	'override-cmd' [2/2]...passed

```

**Pull request merged**